agent/stats/task_windows.go

```diff
@@ -0,0 +1,179 @@
+//go:build windows
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package stats
+
+import (
+	"context"
+	"os/exec"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/aws/amazon-ecs-agent/agent/api/task"
+	"github.com/aws/amazon-ecs-agent/agent/stats/resolver"
+
+	dockerstats "github.com/docker/docker/api/types"
+	"github.com/pkg/errors"
+)
+
+// Making it visible for unit testing
+var execCommand = exec.Command
+
+type StatsTask struct {
+	*statsTaskCommon
+}
+
+func newStatsTaskContainer(taskARN string, containerPID string, numberOfContainers int,
+	resolver resolver.ContainerMetadataResolver, publishInterval time.Duration, taskENIs task.TaskENIs) (*StatsTask, error) {
+	ctx, cancel := context.WithCancel(context.Background())
+
+	devices := make([]string, len(taskENIs))
+	for count, device := range taskENIs {
```

Comment:
`count` threw me off here. Its really `index`

Reply:
Made changes for this one.